src/store/store.ts

```diff
@@ -0,0 +1,21 @@
+import { configureStore } from '@reduxjs/toolkit';
+import createSagaMiddleware from 'redux-saga';
+
+import { movieReducer } from './movie.slice';
+import { rootSaga } from './saga';
+
+const sagaMiddleware = createSagaMiddleware();
+
+export const store = configureStore({
```

Comment:
May include here conditionally (for dev build only) [redux DevTools](https://redux.js.org/usage/configuring-your-store#integrating-the-devtools-extension) which may be quite helpful for debugging purposes.